resources/dicts/events/death/general.json

```diff
@@ -2727,72 +2779,83 @@
                     "m_c"
                 ],
                 "values": [
-                    "dislike",
-                    "platonic"
+                    "dislike"
+                ],
+                "amount": 15
+            },
+            {
+                "cats_from": [
+                    "r_c"
+                ],
+                "cats_to": [
+                    "m_c"
                 ],
-                "amount": -5
+                "values": [
+                    "platonic",
+                    "trust",
+                    "comfort"
+                ],
+                "amount": -15
             }
         ]
     },
     {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_war_any3",
+        "biome": ["any"],
         "camp": [
             "any"
         ],
         "season": [
             "any"
         ],
-        "tags": [
-            "war"
-        ],
+        "sub_type": "war",
+        "tags": [ "some_lives"],
         "weight": 20,
         "event_text": "m_c was killed by a falling boulder pushed from a cliff above {PRONOUN/m_c/object} by o_c warriors.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [ "adolescent", "adult", "young adult", "senior adult" ],
+            "status": [ "apprentice", "warrior", "deputy", "leader" ],
             "dies": true
         },
         "history": {
+            "cats": "m_c",
             "reg_death": "m_c was crushed by a falling boulder.",
             "lead_death": "{VERB/m_c/were/was} crushed by a falling boulder."
         }
     },
     {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_hawk_any1",
+        "biome": ["any"],
         "camp": [
             "any"
         ],
         "season": [
             "any"
         ],
-        "tags": [],
+        "tags": [ "all_lives", "no_body" ],
         "weight": 20,
         "event_text": "r_c, guarding camp, yowls a warning as they spot a circling shadow, right below m_c. r_c starts sprinting to m_c's position, but couldn't get there in time, not before m_c was carried high into the sky.",
         "m_c": {
-            "age": [],
-            "status": [],
-            "relationship_status": [],
+            "age": [ "any" ],
```

Comment:
let's make this kittten and adol only, full grown cats are a bit large for hawks to fly with